TeamCode/src/main/java/org/firstinspires/ftc/teamcode/tutorials/TissueBoxExample.java

```diff
@@ -0,0 +1,27 @@
+package org.firstinspires.ftc.teamcode.tutorials;
+
+public class TissueBoxExample {
+    int MAX_TISSUES = 200;
+    int currentTissueCount;
+
+    public void init() {
+        MAX_TISSUES = 200;
+        currentTissueCount = 200;
+    }
+
+    public String getTissue() {
+        if (currentTissueCount <= 0) {
+            return "no more tissues!";
+        }
+        decrementTissues();
+        return "tissue";
+    }
+
+    public double getPercentFull() {
+        return (double)currentTissueCount / (double)MAX_TISSUES;
+    }
+
+    private void decrementTissues() {
+        currentTissueCount = currentTissueCount - 2;
```

Comment:
I thought it was supposed to be decreased by three! Also the rest of this code is so good i'll need a tissue🥲

Reply:
Fixed it!